scvi/hub/model_card_template.py

```diff
@@ -0,0 +1,45 @@
+template = """\
+---
+{card_data}
+---
+
+# Description
+
+{description}
+
+# Model properties
```

Comment:
were we still going to add the italics info disclaimer about editing certain parts of the model card?

Reply:
I ended up decoupling the model card completely from the required metadata. The latter might be a subset of the former or grow to be a superset or it might just have no relationship to it. Conceptually these are two different things. One -- the model card -- is the human-readable human0facing description of the model. The other -- the metadata -- is some required bits of info needed for scvi-hub to work. There are essentially three options:
1. Embed the metadata in the model card, and ask users not to edit those parts. Essentially this constraints the human-facing contents of the model card. And for that reason, I dont love it.
2. Embed the metadata either in the header or in a commented section at the bottom of the model card. This could work but I prefer 3.
3. What I did, aka decouple the model card completely from the metadata. This makes sense conceptually, is the least error prone in terms of accidental alteration, and avoid having to parse text. Lastly, if the user wants, they can have no model card at all. That's allowed on huggingface.
Happy to chat more today.